src/modules/git_status.rs

```diff
@@ -52,9 +56,6 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         log::trace!("No stash object found");
     }
 
-    let repo_status = get_repo_status(&repository);
-    log::debug!("Repo status: {:?}", repo_status);
-
```

Comment:
I had to move this up to the first part in the file because of the immutable borrows above it. Please let me know if there's a better way to go about this.